src/main/java/ZooPackage/Giraffe.java

```diff
@@ -1,6 +1,6 @@
 package ZooPackage;
 
-public class Giraffe extends Animals{
+public class Giraffe extends Animals{//used inheritance to get methods breathing() and moving()
     @Override
     public void breathing() {
```

Comment:
No need to override the super class methods if you intend to use them directly anyways.
Applies here and in all other heir-classes